### PR TITLE
Fix zero data showing +100% percentage.

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue #4255

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/util/chart.js [hunk 1]
```diff
@@ -73,3 +73,25 @@ export const isSingleSlice = ( report ) => {
...
 	return false;
 };
+
+/**
+ * Calculates difference between two values.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} currentValue  The difference which can be positive or negative.
+ * @param {number} previousValue Options to be passed to the ChangeArrow component.
```
Comment (eugene-manuilov) [inline assets/js/util/chart.js:83]: @kuasha420 these descriptions look irrelevant to what these parameters are. Can you adjust it?

File: assets/js/util/chart.js [hunk 2]
```diff
@@ -73,3 +73,25 @@ export const isSingleSlice = ( report ) => {
...
+ * @return {number} The difference.
+ */
+export const calculateDifference = ( currentValue, previousValue ) => {
```
Comment (eugene-manuilov) [inline assets/js/util/chart.js:86]: Let's probably rename this function to be more specific. Something like `calculateDifferenceBetweenChartValues` will better explain what the function does. What do you think?

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @kuasha420. Almost looks good to me. Added two small nitpicks. Once they are fixed, I'll approve this PR.

File: assets/js/util/chart.js (hunk 2 above)
Comment (kuasha420) [reply assets/js/util/chart.js]: Yes, it makes more sense and will communicate the fact that it's specifically for finding difference in chart value and not a general solution. Updated.

**Review by eugene-manuilov — APPROVED** [review]
Awesome! Thanks, @kuasha420.
